File: app/src/main/java/org/dhis2/usescases/splash/SplashComponent.java
package org.dhis2.usescases.splash;


import org.dhis2.data.dagger.PerActivity;

import dagger.Subcomponent;

@PerActivity
@Subcomponent(modules = SplashModule.class)
public interface SplashComponent {
    void inject(SplashActivity splashActivity);
}

File: app/src/main/java/org/dhis2/usescases/splash/SplashComponent.kt
package org.dhis2.usescases.splash

import dagger.Subcomponent
import org.dhis2.commons.di.dagger.PerActivity

@PerActivity
@Subcomponent(modules = [SplashModule::class])
interface SplashComponent {
    fun inject(splashActivity: SplashActivity)
}
